resources/zib/zib-AdvanceDirective.xml

```diff
@@ -0,0 +1,217 @@
+      <mapping>
+        <identity value="zib-advancedirective-v3.1.1-2020EN" />
+        <map value="NL-CM:7.15.7" />
+        <comment value="LivingWillDate (implicit, main mapping is on `Consent.dateTime`)" />
+      </mapping>
+    </element>
+    <element id="Consent.scope">
+      <path value="Consent.scope" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://terminology.hl7.org/CodeSystem/consentscope" />
+          <code value="adr" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Consent.category">
+      <path value="Consent.category" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://terminology.hl7.org/CodeSystem/consentcategorycodes" />
+          <code value="acd" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Consent.dateTime">
+      <path value="Consent.dateTime" />
+      <short value="LivingWillDate" />
+      <definition value="The date on which the living will was recorded." />
+      <alias value="WilsverklaringDatum" />
+      <mapping>
+        <identity value="zib-advancedirective-v3.1.1-2020EN" />
+        <map value="NL-CM:7.15.7" />
+        <comment value="LivingWillDate" />
+      </mapping>
+    </element>
+    <element id="Consent.source[x]">
+      <path value="Consent.source[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Consent.source[x]:sourceAttachment">
+      <path value="Consent.source[x]" />
+      <sliceName value="sourceAttachment" />
+      <type>
+        <code value="Attachment" />
+      </type>
+    </element>
+    <element id="Consent.source[x]:sourceAttachment.data">
+      <path value="Consent.source[x].data" />
+      <short value="LivingWillDocument" />
+      <definition value="Scanned source document with the living will and the patient's signature, such as a PDF." />
+      <alias value="WilsverklaringDocument" />
+      <mapping>
+        <identity value="zib-advancedirective-v3.1.1-2020EN" />
+        <map value="NL-CM:7.15.5" />
+        <comment value="LivingWillDocument" />
+      </mapping>
+    </element>
+    <element id="Consent.policy.uri">
+      <path value="Consent.policy.uri" />
+      <comment value="Either a policy or policyRule is mandated by the HL7 core resource by the ppc-1 constraint, therefore, if no explicit value is recorded a fallback is to populate `policy.uri` with _http://wetten.overheid.nl/_." />

```

Comment:
Ik zou het iets anders verwoorden, met toch ook de belangrijke nuance dat de suggestie alleen van toepassing is als ook daadwerkelijk de Nederlandse wet wordt gevolgd.

> Either `.policy` or `.policyRule` should exist following core constraint ppc-1. If no explicit value is recorded, a fallback is to populate `policy.uri` with 'https://wetten.overheid.nl/' if the advance directive is recorded following Dutch law and regulations.

Reply:
Nice! Aangepast.